tests/test_unix.py

```diff
@@ -7,15 +7,46 @@
 from _pytest.monkeypatch import MonkeyPatch
 from pytest_mock import MockerFixture
 
+import platformdirs.unix
 from platformdirs.unix import Unix
 
 
+def test_user_documents_dir(monkeypatch: MonkeyPatch) -> None:
+    example_path = "/home/example/ExampleDocumentsFolder"
+    monkeypatch.setattr(platformdirs.unix, "get_user_dirs_folder", lambda key: example_path)
```

Comment:
No need to import it, can mock without it.

Reply:
Can you say what doesn't need to be imported and a more detailed why? I'm not seeing it

Reply:
`platformdirs.unix`. Just less code to maintain.